governance.md

```diff
@@ -0,0 +1,137 @@
+- GitHub repository hosting.
+
+Being an Owner is not time-limited. There is no fixed size of the Owner group. The Owner group should be of such a size as to ensure adequate coverage of important areas of expertise balanced with the ability to make decisions efficiently.
+An Owner may be removed from the Owner group by voluntary resignation, or by a standard Owner group motion.
+
+Changes to the Owner group should be posted in the agenda, and may be suggested as any other agenda item (see [Project Meetings](#project-meetings) below).
+
+Owners have additional responsibilities over and above those of a Peer. These responsibilities ensure the smooth running of the project. Owners are expected to review code contributions, approve changes to this document, manage the copyrights within the project outputs, and participate in the project discussions and meetings.
+
+Owners fulfill all requirements of Peers, and also:
+
+- Manage and merge non-data pull requests. Such as schema, linter or infrastructure changes.
+- May merge their own pull requests once they have collected the feedback they deem necessary. (No pull request should be merged without at least one peer or owner comment stating they’ve looked at the PR.)
+- Release a new npm version of the project on a regular (weekly) basis.
+
+To become an Owner one must fulfill at least the following items and commit to being a part of the community for the long-term.
+
+- Work in a helpful and collaborative way with the MDN community.
+- Have given good feedback on others’ submissions and displayed an overall understanding of the code quality standards for the project.
+- Ability to drive the project forward, manage requirements from users, and taking on responsibility for the overall health of the project.
+
+An individual is invited to become an Owner by existing Owners. A nomination will result in discussion and then a decision by the Owner group.
+
+#### List of Owners
+- Florian Scholz (@Elchi3), Mozilla, BCD project lead
+- Will Bamberg (@wbamberg), Mozilla
+- Chris David Mills (@chrisdavidmills), Mozilla
+- Kadir Topal (@atopal), Mozilla
+
+The owners are also listed as top-level owners in GitHub’s code owner file.
+
+## Project Meetings
+There are no recurrent project meetings, they are scheduled when required, at a time that works for the owners, and using tools that enable participation by the community. The meeting is run by a designated moderator approved by the owners group.
+Items are added to the agenda which are considered contentious or are modifications of governance, contribution policy, owner membership, or release process.
+
+The intention of the agenda is not to approve or review all patches. That should happen continuously on GitHub and be handled by the larger group of Peers. The exception to this is when defining how the schema should look like (or when proposing an update), there is not an initial agreement in the GitHub discussion, and action needs to be taken.
```

Comment:
"update), there is not an initial agreement in the GitHub discussion, and action needs to be taken." -> I am a bit confused by the structure here. Should it be something like "update), or when a PR discussion has stalled due to disagreement or inaction, and progress needs to be unblocked."